pkg/cli/iam_handle.go

```diff
@@ -178,7 +192,20 @@ func (c *IAMHandleCommand) handleIAM(ctx context.Context) error {
 	if _, err := h.Do(ctx, reqWrapper); err != nil {
 		return fmt.Errorf("failed to handle IAM request: %w", err)
 	}
-	c.Outf("Successfully handled IAM request")
+
+	if c.flagVerbose {
+		enc := yaml.NewEncoder(c.Stdout())
```

Comment:
Since this is repeated in multiple commands, and since it could use some dedicated unit tests, I'd recommend making a helper like `encodeYAML(w io.Writer)` that encapsulates this logic for the cli. It looks like you already kind of did that with `output`, but it's a slightly different API.

Reply:
Sure, updating.

Reply:
Add helper in util.go, I wonder if we need dedicated unit tests for private helpers?